red/My-List-Util/t/has_hosts.t

```diff
@@ -0,0 +1,25 @@
+use Test::More;
+use Test::File;
+
+my $nix_path = '/etc/hosts';
+my $windows_path = 'C:\Windows\System32\drivers\etc\hosts';
+
+my %supported_unixlike = map { $_ => 1 } qw( bsdos darwin linux openbsd freebsd solaris );
```

Comment:
Nope, just Linux. Didn't want to spend the evening configuring VMs for everything else.